### DISCUSSION TITLE
How to move my pro team space projects to my hobby space?

### DISCUSSION THREAD

**Discussion by markbudada** [vercel/community#4917] [Help] [answered]
### Summary

Because I found that I don't need a pro now, so I decide to cancel it, and seems like the 1st step is to move my projects into my hobby space.

But I found that I don't have a hobby space, maybe because I upgrade to my hobby space to the pro team space directly so it disappear?

### Steps to Reproduce

1. Create a vercel account
2. Deploy a simple project that fit into the hobby plan ([Blog](https://vercel.com/templates/next.js/blog-starter-kit))
3. Try to deploy a template that need Pro's ability ([the AI chatbot](https://github.com/vercel-labs/ai-chatbot))
4. You will get deploy error and it prompts you to upgrade to pro
5. After upgraded, these 2 projects are both in the Pro team, and the hobby space disappear

**Comment by swarnava** [comment, marked as answer]
Hi there,

You can create a Pro trial team and let it expire which will eventually become Hobby team. You can learn more about what you can host on Hobby plan here which includes any production usecase: https://vercel.com/docs/concepts/limits/fair-use-policy
